<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPS3DCapturePluginDialogBase</class>
 <widget class="QDialog" name="GPS3DCapturePluginDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPS 3D Capture Plugin</string>
  </property>
  <widget class="QPushButton" name="asciiFilePushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Ascii File:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="asciiFileLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>681</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="crsPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select CRS:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="crsLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="geoidCheckBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract an interpolated geoid height</string>
   </property>
  </widget>
  <widget class="QLabel" name="geoidLabel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Geoid Model:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="geoidComboBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
